setup.py

```diff
@@ -64,6 +64,7 @@ def exec_file(path_segments):
         "pytz==2018.9",
         "requests==2.21.0",
         "python-dateutil==2.8.1",
+        "asyncio==3.4.3",
```

Comment:
Not needed. The asyncio module is part of the Python standard library since Python 3.4

Reply:
Done.